packages/components/gulpfile.js

```diff
@@ -10,6 +10,9 @@ const sass = require('gulp-sass');
 const postcss = require('gulp-postcss');
 const autoprefixer = require('autoprefixer');
 const deduper = require('postcss-discard-duplicates');
+const Fiber = require('fibers');
+
+sass.compiler = require('sass');
```

Comment:
totally can revert this set of changes :+1:

Reply:
Thank you for leaving a note here @cal-smith! Would you be able to add an option to [`cloptions`](https://github.com/carbon-design-system/carbon/blob/v10.3.0/packages/components/gulpfile.js#L69-L92) and assign `sass.compiler` if such option is enabled? (Note: Please make sure the short version doesn't collide with one for Gulp itself)

Reply:
Sure 👍 should I default it to `node-sass` or `sass`?

Reply:
Let's default to `node-sass` for now. It's just about the balance between existing applications using `node-sass` vs. new/modified applications using Dart Sass, though, and we'd flip the switch in next major release.

Reply:
Sure, makes sense :+1: